src/main/java/collab/DataBase.java

```diff
@@ -69,8 +69,8 @@ public Employee deleteItem(Employee item) {
 
   public Employee deleteItemById(String id) {
     List<Employee> foundItems = deleteItemByCondition("employeeNum", id);
-    if(foundItems.size()==0) return null;
-    return deleteItemByCondition("employeeNum", id).get(0);
+    if(foundItems.isEmpty()) return null;
+    return foundItems("employeeNum", id).get(0);
```

Comment:
이 부분 잘못된 것 같습니다. 변수인데 함수처럼 사용된 것 같아요.
아래처럼 사용하는 것이 맞지 않을까 합니다.

```suggestion
    return foundItems.get(0);
```

Reply:
넵. 거의 다 된 것 같습니다!!